frontend/e2e/pages/patients.js

```diff
@@ -29,6 +31,9 @@ function addPatient(dobFormat) {
   this.section.editPatient.setValue("@address", address);
   this.section.editPatient.setValue("@state", state);
   this.section.editPatient.setValue("@zip", zip);
+  this.section.editPatient.setValue("@role", role);
+  this.expect.section("@editPatient").to.contain.text("Student ID");
```

Comment:
Oddly this seems to fail during CI but passes locally (using Chrome). Does anyone know what's going on here? The field _should_ dynamically appear after the role is set to `"STUDENT"`. I'm assuming `this.expect` has some retry interval and this is not a timing thing but I'm not sure.